catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/reader/TransactionMessageBodyReaderTest.java

```diff
@@ -104,7 +104,7 @@
   private static final String DELETE_REQUEST_CQL_XML =
       "<csw:Transaction service=\"CSW\"\n"
           + "   version=\"2.0.2\" xmlns:csw=\"http://www.opengis.net/cat/csw/2.0.2\"\n"
-          + "   xmlns:ogc=\"http://www.opengis.net/ogc\">\n"
+          + "   xmlns:ogc=\"http://www.opengis.net/cat/csw/2.0.2\">\n"
```

Comment:
why are we changing this namespace?

Reply:
There were NPEs in for all tests that checked the value `cqlText` field. The field was null, and I traced it back to a JAXB unmarshalling exception that was swallowed. The exception said `cqlText` was an unexpected element, and the expected elements were either the filter field from the original namespace, or the cqlText field from this extended namespace. I gave ogc the extended namespace and all the tests pass.

If this doesn't make sense, or there is a reason this should remain as is, please let me know and I can research an alternative solution.

Reply:
Looks like it is part of the CSW namespace http://schemas.opengis.net/csw/2.0.2/CSW-discovery.xsd.  I think this makes sense. 👍